src/main/java/net/dv8tion/jda/api/entities/Invite.java

```diff
@@ -310,12 +310,17 @@ default String getUrl()
     boolean isExpanded();
 
     /**
-     * Whether this Invite grants only temporary access or not
+     * Whether this Invite grants only temporary access or not.
+     *
+     * <p>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
      *
      * @throws IllegalStateException
      *         if this invite is not expanded
      *
-     * @return Whether is invite expanded or not
+     * @return Whether is invite temporary or not
```

Comment:
Let's use this chance to fix the grammar as well:

```suggestion
     * @return Whether this invite temporary or not
```

Feel free to fix it everywhere else in the Invite docs.